lib/fluffle/handlers/base.rb

```diff
@@ -4,6 +4,12 @@ class Base
       def call(method:, params:, id:, meta:)
         raise RuntimeError, '#call is not implemented on abstract Base handler class'
       end
+
+      # This is called *after* the server has published the response message
+      # to the client's queue.
+      def after_response(request:)
```

Comment:
Isn't this problematic as it will cause `respond_to?` to return `true` for anything inheriting the base handler, whether or not they implement their own logic?